crates/indexer/src/processors/default_processor.rs

```diff
@@ -151,7 +151,20 @@ fn insert_user_transactions_w_sigs(
             diesel::insert_into(schema::user_transactions::table)
                 .values(&all_user_transactions[start_ind..end_ind])
                 .on_conflict(ut_schema::version)
-                .do_nothing(),
+                .do_update()
+                .set((
+                    ut_schema::block_height.eq(excluded(ut_schema::block_height)),
+                    ut_schema::parent_signature_type.eq(excluded(ut_schema::parent_signature_type)),
+                    ut_schema::sender.eq(excluded(ut_schema::sender)),
+                    ut_schema::sequence_number.eq(excluded(ut_schema::sequence_number)),
+                    ut_schema::max_gas_amount.eq(excluded(ut_schema::max_gas_amount)),
+                    ut_schema::expiration_timestamp_secs
+                        .eq(excluded(ut_schema::expiration_timestamp_secs)),
+                    ut_schema::gas_unit_price.eq(excluded(ut_schema::gas_unit_price)),
+                    ut_schema::timestamp.eq(excluded(ut_schema::timestamp)),
+                    ut_schema::entry_function_id_str.eq(excluded(ut_schema::entry_function_id_str)),
+                    ut_schema::inserted_at.eq(excluded(ut_schema::inserted_at)),
+                )),
```

Comment:
why?

Reply:
so that i can override previously bad results? I removed this before b/c i guess results shouldn't change but it is useful in case we need to fix something.